### PR TITLE
Add ties breaks and multi-winners

### PR DESCRIPTION
**Pull request by pkoch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,23 @@ There will be prizes for the three winning team in each category. We still haven
 Each category has their own independent voting system. This means that voting in one category in no way influences the winners of the other categories.
 For each category, voters are allowed to vote in three projects, in order of preference. When the ballots close, we sort the projects by their ranking, where ranking can be determined by [Schulze method](https://en.wikipedia.org/wiki/Schulze_method).
 
+#### Breaking ties
+
+Breaking ties in the Schulze method takes various forms, each with their pros
+and cons. We've decided to keep this simple and break ties randomly, since all
+the winners are equally preferred by the audience. Before the voting begins,
+we'll publicaly assign each team a "tie breaking dice roll", and use that in
```
Comment (jpmcosta) [inline README.md]: publicly*

File: README.md [hunk 2]
```diff
@@ -139,6 +139,23 @@ There will be prizes for the three winning team in each category. We still haven
...
+case we have to.
+
+#### Winning multiple categories
+
+In order to avoid concentrating the spoils on a particularly successful
+project, we'll only award one prize per team. If a team wins multiple
+categories, they'll be given the choice of which prize to win.
```
Comment (jpmcosta) [inline README.md]: Maybe something like:
```
If a team wins multiple categories, they'll win their preferred prize, and the second best teams in the other categories will take the remaining prizes.
```

**Review by jgradim — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (pkoch) [reply README.md]: Maybe `s/prize/category/`? I've tried to avoid using "prizes" on purpose.

File: README.md (hunk 1 above)
Comment (pkoch) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (pkoch) [reply README.md]: We talked, I dropped the `prizes` opposition.

Comment (pkoch) [reply README.md]: Leão said yes.

**Pull request merged**